scst-store/tls-configuration.hbs.md

```diff
@@ -0,0 +1,53 @@
+
+# Setting up TLS in Metadata Store
+
+###  <a id="Setting up tls namespace and secretName"></a>Setting up tls namespace and secretName
+
+In `tap-values` file you can set namespace and secretName as shown below.
```

Comment:
We typically refer to the file as `tap-values.yaml` in other parts of the docs. 

I also think we should add the code backticks to `namespace` and `secretName`.